physx-sys/src/physx_api.cpp

```diff
@@ -294,27 +350,53 @@ extern "C"
     }
 
     PxRaycastCallback *create_raycast_callback(
-        HitProcessTouchesCallback process_touches_callback,
+        RaycastHitProcessTouchesCallback process_touches_callback,
         HitFinalizeQueryCallback finalize_query_callback,
+        PxRaycastHit *touchesBuffer,
+        PxU32 numTouches,
         void *userdata
     ) {
-		return new HitCallbackTrampoline<PxRaycastHit>(process_touches_callback, finalize_query_callback, userdata);
+        return new RaycastHitCallbackTrampoline(
+            process_touches_callback, finalize_query_callback, touchesBuffer, numTouches, userdata);
+    }
+
+    void delete_raycast_callback(PxRaycastCallback *callback)
+    {
+        delete callback;
+    }
```

Comment:
Should be a `get_raycast_callback_user_data` here, otherwise it's irretrievable from Rust and it'll get leaked on drop.